<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncoderApp</class>
 <widget class="QMainWindow" name="EncoderApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encrypt &amp; Decrypt App</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">#title_Encoder {
	font-size: 24px;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="title_Encoder">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Encrypt&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_einput">
           <property name="text">
            <string>Input:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_en_loadFile">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 80, 80);</string>
             </property>
             <property name="text">
              <string>Load File</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_en_enterText">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 80, 80);</string>
             </property>
             <property name="text">
              <string>Enter Text</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_en_preview">
           <property name="text">
            <string>Preview</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_esettings">
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="comboBox_en_type"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">color: black;</string>
             </property>
             <property name="text">
              <string>Method encrypting</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QSpinBox" name="spinBox_ca_key">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_ca_key">
             <property name="styleSheet">
              <string notr="true">color: black;</string>
             </property>
             <property name="text">
              <string>Encrypting Key</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_en_inv_sentence">
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Invert simple words?</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_en_dec_space">
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Use '-' separator?</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_en_bin_enter">
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Include enter key?</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_en_bin_spaces">
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Insert space every 8 bits?</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_en_code">
         <property name="text">
          <string>ENCRYPT!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="title_Decoder">
         <property name="font">
          <font/>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Decrypt&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_dinput">
           <property name="text">
            <string>Input:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_de_load">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 80, 80);</string>
             </property>
             <property name="text">
              <string>Load File</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_de_enterText">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 80, 80);</string>
             </property>
             <property name="text">
              <string>Enter Text</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="btn_en_preview_2">
           <property name="text">
            <string>Preview</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_dprocess">
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="comboBox_de_type"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Type of Decrypting</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_de_code">
         <property name="text">
          <string>DECRYPT!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
